### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by yossifirouz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by AradCarmi** [comment]
Hey @yossifirouz ,
Thank you for your contribution!

In order to start my review, I will kindly ask you to fill out the [contribution form](https://forms.gle/XDfxU4E61ZwEESSMA)

Please don't hesitate to contact me with any questions :)

File: Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules.xif [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+| alter Process_id= arrayindex(regextract(_raw_log ,"PID\s(\d+)") ,0)
+| alter Command= arrayindex(regextract(_raw_log ,"command:\s\'?([^\'?\"]+)") ,0)
+| alter destination_ip = arrayindex(regextract(_raw_log ,"on\s(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})") ,0)
+| alter source_ip1= arrayindex(regextract(_raw_log ,"from\s(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})") ,0)
+| alter source_ip2= arrayindex(regextract(_raw_log ,"Source\sIP\:\s(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})") ,0)
+| alter sourceip  = coalesce(source_ip2, source_ip1 ) 
+| alter appliance= arrayindex(regextract(_raw_log ,"Appliance:\s([^\,]+)") ,0)
+| alter location= arrayindex(regextract(_raw_log ,"Location\:\s([^\,]+)") ,0)
+| alter event_id= arrayindex(regextract(_raw_log ,"Event\:\s([^\.]+)") ,0)
+
```
Comment (evisochek) [inline Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules.xif:52]: please remove the extra white space

File: Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules.xif [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+| alter XDM.Auth.observation_timestamp =parse_timestamp("%h %e %T",Timestamp)
+| alter XDM.Auth.original_event_description = Message
+| alter XDM.Auth.original_event_type = log_category
+| alter XDM.Auth.Client.user.username= Username
+| alter XDM.Auth.Client.process.pid= Process_id
+| alter XDM.Auth.Server.process.command_line= command 
+| alter XDM.Auth.Client.host.ipv4_addresses = Destination_ip
```
Comment (evisochek) [inline Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules.xif:59]: data model validation is failing - expecting array but receiving a string

File: Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules.xif [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+| alter sourceip  = coalesce(source_ip2, source_ip1 ) 
+| alter appliance= arrayindex(regextract(_raw_log ,"Appliance:\s([^\,]+)") ,0)
+| alter location= arrayindex(regextract(_raw_log ,"Location\:\s([^\,]+)") ,0)
+| alter event_id= arrayindex(regextract(_raw_log ,"Event\:\s([^\.]+)") ,0)
+
+| alter XDM.Auth.observation_timestamp =parse_timestamp("%h %e %T",Timestamp)
+| alter XDM.Auth.original_event_description = Message
+| alter XDM.Auth.original_event_type = log_category
+| alter XDM.Auth.Client.user.username= Username
+| alter XDM.Auth.Client.process.pid= Process_id
```
Comment (evisochek) [inline Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules.xif:57]: data model validation is failing - expecting number but receiving a string

**Review by evisochek — APPROVED** [review]

File: Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules_schema.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "Cisco_SMA_raw": {
+      "_raw_log": {
+        "type": "string",
+        "is_array": false
+      },
```
Comment (evisochek) [inline Packs/CiscoSMA/ModelingRules/CiscoSMAModelingRules/CiscoSMAModelingRules_schema.json:6]: extra ",", please remove

**Pull request merged**